src/OpenTelemetry.Resources.Gcp/OpenTelemetry.Resources.Gcp.csproj

```diff
@@ -15,9 +15,9 @@
   <ItemGroup>
     <PackageReference Include="Google.Api.Gax" Version="4.8.0" />
     <PackageReference Include="OpenTelemetry" Version="$(OpenTelemetryCoreLatestVersion)" />
-    <PackageReference Include="System.Text.Json" Version="4.7.2" />
+    <PackageReference Include="System.Text.Json" Version="$(SystemTextJsonPkgVer)" />
     <!-- System.Text.Encodings.Web is indirect reference. It is needed to upgrade it directly to avoid https://github.com/advisories/GHSA-ghhp-997w-qr28 -->
-    <PackageReference Include="System.Text.Encodings.Web" Version="4.7.2" />
+    <PackageReference Include="System.Text.Encodings.Web" Version="$(SystemTextEncodingsWebPkgVer)" />
```

Comment:
You can remove the `System.Text.Encodings.Web` reference where you have a direct reference to `System.Text.Json` v8.0.4. The reason `System.Text.Encodings.Web` was here is the System.Text.Json v4.7.2 requires `System.Text.Encodings.Web` >= 4.7.1 but `System.Text.Encodings.Web` v4.7.1 is vulnerable so you have to explicitly add a reference to v4.7.2.

When you reference `System.Text.Json` v8.0.4 you get `System.Text.Encodings.Web` v8.0.0 transitively and it is safe (for the moment) 😏

Reply:
I didn't notice `System.Text.Json` depends on `System.Text.Encodings.Web`. I removed direct reference to `System.Text.Encodings.Web` in this case.